### PR TITLE
Sidebar logo fixed in all the pages

### LINKED ISSUE #236: Issue with Sidebar button in some pages
- [x] [Interactive Karnaugh Map](https://learn.circuitverse.org/docs/kmapi.html) - Doesn't have a sidebar button. 
- [ ] [Half Adder](https://learn.circuitverse.org/docs/Combinational/half_adder.html) - Doesn't have the icon in sidebar button.

### PR DESCRIPTION
**Pull request by abhinavsri360**
A meaningful title for PR, not like: made changes to xyz.md

Fixes #
There was an issue with the sidebar button in some pages i.e., the button was not uniform across all pages. Though I found an alternative way to fix that still I would like to bring to everyone's notice that when the pages with children are loaded they throw an error of something like this:
image
But pages work fine and Sidebar logo is fixed in all the pages even in the page 404
Closes #236
Ref #

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
There is an inactive PR present #237 
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add a link from the checks tab after checks complete)
https://deploy-preview-346--tender-ardinghelli-ebb79f.netlify.app/docs/combinational
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by abhinavsri360** [comment]
I would request all of you to review this:
@YashKumarVerma @ShubhankarKG @Shivansh2407 
@Shivansh2407 this works for the page 404 as well.

File: _includes/head.html [hunk 1]
```diff
@@ -31,8 +31,17 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   
-  <script defer src="../assets/js/fontawesome-solid.js"></script>
-  <script defer src="../assets/js/fontawesome-main.js"></script>
+    
+  {% if page.parent %}
+    <script defer src="../../assets/js/fontawesome-solid.js"></script>
+    <script defer src="../../assets/js/fontawesome-main.js"></script>
+  {% else if parent %}
```
Comment (ShubhankarKG) [inline _includes/head.html:38]: What does parent refer to ? All page attributes are available via **page.something** syntax. Might consider removing this.

**Comment by abhinavsri360** [comment]
parent refers to the file something like flipflop/flip-flops.md here flipflop is not in docs directory also it is not even a child page that's why parent refers to the page which contains all child's. Removing this line will cause sidebar logo differences in parent pages, you can try removing this line in your local machine and navigate to msi components page sidebar logo will differ from what it should be that's why that line is there hope you get that 😅

**Comment by ShubhankarKG** [comment]
I checked it locally, and it seems that the sidebar displays properly even when parent clause is absent, in all the pages. You might wanna remove it.

**Comment by Shivansh2407** [comment]
> parent refers to the file something like flipflop/flip-flops.md here flipflop is not in docs directory also it is not even a child page that's why parent refers to the page which contains all child's. Removing this line will cause sidebar logo differences in parent pages, you can try removing this line in your local machine and navigate to msi components page sidebar logo will differ from what it should be that's why that line is there hope you get that

It would be better if you can post a screenshot for this situation. Rest all looks good to me. Nice work @abhinavsri360 👍

**Comment by abhinavsri360** [comment]
> > parent refers to the file something like flipflop/flip-flops.md here flipflop is not in docs directory also it is not even a child page that's why parent refers to the page which contains all child's. Removing this line will cause sidebar logo differences in parent pages, you can try removing this line in your local machine and navigate to msi components page sidebar logo will differ from what it should be that's why that line is there hope you get that
> 
> It would be better if you can post a screenshot for this situation. Rest all looks good to me. Nice work @abhinavsri360 👍

image
@Shivansh2407 Have a look at what happens when we remove that line
@ShubhankarKG It do makes a difference

**Pull request merged**